install/Knative-with-Minikube.md

```diff
@@ -123,12 +123,22 @@ If you'd like to view the available sample apps and deploy one of your choosing,
 head to the [sample apps](../serving/samples/README.md) repo.
 
 > Note: When looking up the IP address to use for accessing your app, you need
-> to look up the NodePort for the `knative-ingressgateway` as well as the IP
+> to look up the NodePort for the ingress gateway service well as the IP
 > address used for Minikube. You can use the following command to look up the
 > value to use for the {IP_ADDRESS} placeholder used in the samples:
 
 ```shell
-echo $(minikube ip):$(kubectl get svc knative-ingressgateway --namespace istio-system --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')
+# In Knative 0.2.x or prior versions, we use `knative-ingressgateway`.
+INGRESSGATEWAY=knative-ingressgateway
+
+# In Knative 0.3.x the use of `knative-ingressgateway` is deprecated.
+# We should use `istio-ingressgateway` since `knative-ingressgateway`
```

Comment:
GLOBAL. Please change:
```
# In Knative 0.3.x the use of `knative-ingressgateway` is deprecated.
# We should use `istio-ingressgateway` since `knative-ingressgateway`
# will be removed in 0.4.
```

to:
```
# The use of `knative-ingressgateway` is deprecated in Knative v0.3.x.
# Use `istio-ingressgateway` instead, since `knative-ingressgateway`
# will be removed in Knative v0.4.
```